zppy/__main__.py

```diff
@@ -0,0 +1,88 @@
+import argparse
+from configobj import ConfigObj
+from validate import Validator
+import errno
+import os
+
+from zppy.climo import climo
+from zppy.ts import ts
+from zppy.e3sm_diags import e3sm_diags
+from zppy.e3sm_diags_vs_model import e3sm_diags_vs_model
+from zppy.amwg import amwg
+from zppy.mpas_analysis import mpas_analysis
+from zppy.global_time_series import global_time_series
+
+def main():
+  
+  # Command line parser
+  parser = argparse.ArgumentParser(description='Launch E3SM post-processing tasks', usage="zppy -c <config>")
+  parser.add_argument(
+    '-c', '--config', type=str, help='configuration file',required=True)
+  args = parser.parse_args()
+
+  # Subdirectory where templates are located
+  templateDir = os.path.join( os.path.dirname(__file__), 'templates' )
```

Comment:
While this is not an ideal practice, this will work for finding `templates` in circumstances we work in.  See https://pyoxidizer.readthedocs.io/en/v0.5.0/packaging_pitfalls.html#reliance-on-file for some pitfalls and why `importlib.resources` would be preferable in general.